cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -437,3 +442,27 @@ func randomizePoolList(list *apis.CStorPoolInstanceList) *apis.CStorPoolInstance
 
 	return res
 }
+
+// CVCWithVersionDetails build cvc resource with required version
+// details and volume references
+func CVCWithVersionDetails(
+	claim *apis.CStorVolumeClaim,
+	volumeRef *corev1.ObjectReference,
+) (*apis.CStorVolumeClaim, error) {
+
+	cvcObj, err := cvc.BuildFrom(claim).
+		WithClaimRef(volumeRef).
+		WithStatusCapacity(claim.Spec.Capacity).
```

Comment:
we are filling capacity as well? aren't we planned it as part of resize code path getting executed during volume creation?

Reply:
Here `status.capacity` is getting populated as part of initial volume creation once the everthing got successfully created, which is helpful in case of resize compare conditions.
With this we will have a full resource instead of having it at the time of resize .